l1-contracts/src/mock/StakingAssetHandler.sol

```diff
@@ -288,25 +290,24 @@ contract StakingAssetHandler is IStakingAssetHandler, Ownable {
     // If active, nullifiers will end up being zero, but it is user provided input, so we are sanity checking it
     require(_params.devMode == false, InvalidProof());
 
+    require(keccak256(bytes(_params.domain)) == keccak256(bytes(validDomain)), InvalidDomain());
```

Comment:
this is not enough to verify the scope

the scopes in the param object need to be linked to the scope in the public inputs to the proof - which are public inputs 10 - 11. The verifyScopes function still exists on the zkpassport verifier

Reply:
my mistake, its inside verifyProof